### DISCUSSION TITLE
Option to turn off JIT compilation warning

### DISCUSSION THREAD

**Discussion by Ghirigoro** [tailwindlabs/tailwindcss#5906] [Help]
I have a build system where builds are run inside of their own node processes and are considered to have failed if something is emitted on `stderr`. Unfortunately the warning about using JIT compilation is emitted on `stderr` and as far as I can tell there's no way to suppress it. Is there a secret option I'm unaware of? If not is it possible to create one?

**Comment by eldarshamukhamedov** [comment]
Opened issue with repro steps:
https://github.com/tailwindlabs/tailwindcss/issues/8050
